protected/humhub/modules/space/Module.php

```diff
@@ -62,8 +62,9 @@ public function getPermissions($contentContainer = null)
         if ($contentContainer instanceof models\Space) {
             return [
                 new permissions\InviteUsers(),
+                new permissions\ViewAboutPage(),
```

Comment:
We do not require a separate permission for the Space About page. 
Space visibility is sufficient here.

Reply:
I mainly did this so that it could be configurable by the space owner/admin but you're right it's not really needed in the long run of things :thinking:

Reply:
Here a checkbox would probably be sufficient. But I would add this later if necessary.

Reply:
I will have to let the HumHub team do this in this case, because I wouldn't begin to know how to do this :sweat_smile: